cognitedata/reveal
# feat: texture rendering improved to match GLTF viewer

#### Type of change

![Feat](https://img.shields.io/badge/Type-Feat-green)  

#### Jira ticket :blue_book:


https://cognitedata.atlassian.net/browse/REV-800

## Description 
![gltf-viewer](https://user-images.githubusercontent.com/87521752/224052797-85eb7bd3-ce16-4a7e-a079-d091bc08188b.png)
![Reveal-WithGammaCorrection](https://user-images.githubusercontent.com/87521752/224052807-f3d78550-4a5c-4f30-8390-6988a20882f1.png)
:pencil:


## How has this been tested? :mag:
In examples

## Test instructions :information_source:
`cd examples && yarn && yarn start`
Load an textured model (check the JIRA for example model)


## Checklist :ballot_box_with_check:
